src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -296,7 +300,11 @@ public static String getListingComments(Appeal appeal, List<CcdValue<OtherParty>
                     .collect(Collectors.toList()));
         }
 
-        return listingComments.isEmpty() ? null : String.join(String.format("%n%n"), listingComments);
+        if (listingComments.isEmpty()) {
+            return null;
+        }
```

Comment:
Why null, not a blank string?
If a blank string, then this if is not relevant too

Reply:
This is an optional field on the request, if there are no comments, we do not want it to appear on the request so need to send null so the field isn't present on the request.